mvpa2/algorithms/group_clusterthr.py

```diff
@@ -457,9 +461,13 @@ def get_thresholding_map(data, p=0.001):
     p_index = int(len(data) * p)
     if p_index < 1:
         raise ValueError("requested probability is too low for the given number of samples")
-    # threshold indices are all in one row of the argsorted inputs
-    thridx = np.argsort(data, axis=0, kind='quicksort')[-p_index]
-    return data[thridx, np.arange(data.shape[1])]
+    if externals.versions['numpy'] >= '1.9.0':
+        return np.percentile(data, q=(1-p)*100, interpolation='higher', axis=0)
```

Comment:
It may be 3 times faster, but (for data this size) in absolute terms it's only 40 ms faster. 
Is there reason to:
- expect data in real-world scenario's to be much bigger than this, leading to a significant larger delay using the original code; and
- assume that, for typical use case scenarios, a considerable amount of the total execution time of the main function is spent in these lines?
If not, I would vote for not including it.

I understand that this is a judgment call choosing between optimising and maintainability, and I appreciate the efforts for optimization; yet the speed benefit seems too small to justify maintaining and testing* two separate implementations.

[*] it would seem that testing two version of numpy would also involve quite a bit of extra effort.